strongbox-storage/strongbox-storage-core/src/main/java/org/carlspring/strongbox/storage/repository/RepositoryPolicyEnum.java

```diff
@@ -25,9 +41,22 @@ public String getPolicy()
         return policy;
     }
 
-    public void setPolicy(String policy)
+    public boolean acceptsReleases()
     {
-        this.policy = policy;
+        return true;
```

Comment:
I'm not sure if I follow this comment. `true` is the default return value for RepositoryPolicyEnum `acceptsReleases() `method of constant values. Most of RepositoryPolicyEnum constant  values (actually 2 of 3) have this return value equal to `true`. That's why the default here is `true`. Would you like me to change it ?